### PR TITLE
Add quirk id for Xiaomi Aqara Vibration AQ1

### PR DESCRIPTION
**Pull request by Caius-Bonus**
## Proposed change

Add quirk id for Xiaomi Aqara Vibration AQ1 to prepare for removal of "Ugly Hack" in zha ProbeEndpoint.
This is necessary to cleanly merge https://github.com/home-assistant/core/pull/101709 .

## Checklist

- [ ] The changes are tested and work correctly
- [x] `pre-commit` checks pass / the code has been formatted using Black
- [ ] Tests have been added to verify that the new code works

### CODE REVIEW

File: zhaquirks/xiaomi/aqara/vibration_aq1.py [hunk 1]
```diff
@@ -69,6 +69,8 @@
 class VibrationAQ1(XiaomiQuickInitDevice):
     """Xiaomi aqara smart motion sensor device."""
 
+    quirk_id = "xiaomi_aqara_vibration_aq1"
```
Comment (TheJulianJES) [inline zhaquirks/xiaomi/aqara/vibration_aq1.py]: Can you add it here https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/quirk_ids.py and then reference the quirk ID?

We'll also need to do a quirks bump and change in HA.

Comment (Caius-Bonus) [reply zhaquirks/xiaomi/aqara/vibration_aq1.py]: Yes, thanks.

File: zhaquirks/quirk_ids.py [hunk 2]
```diff
@@ -3,3 +3,5 @@
 # Tuya
 TUYA_PLUG_ONOFF = "tuya.plug_on_off_attributes"  # plugs with configurable attributes on the OnOff cluster
 TUYA_PLUG_MANUFACTURER = "tuya.plug_manufacturer_attributes"  # plugs with configurable attributes on a custom cluster
+
+XIAOMI_AQARA_VIBRATION_AQ1 = "xiaomi.aqara_vibration_aq1"
```
Comment (TheJulianJES) [inline zhaquirks/quirk_ids.py]:
```suggestion
# Xiaomi
XIAOMI_AQARA_VIBRATION_AQ1 = "xiaomi.aqara_vibration_aq1" # vibration sensor with custom cluster handler
```

File: zhaquirks/quirk_ids.py [hunk 3]
```diff
@@ -3,3 +3,8 @@
 # Tuya
 TUYA_PLUG_ONOFF = "tuya.plug_on_off_attributes"  # plugs with configurable attributes on the OnOff cluster
 TUYA_PLUG_MANUFACTURER = "tuya.plug_manufacturer_attributes"  # plugs with configurable attributes on a custom cluster
+
+# Xiaomi
+XIAOMI_AQARA_VIBRATION_AQ1 = (
+    "xiaomi.aqara_vibration_aq1"  # vibration sensor with custom cluster handler
+)
```
Comment (TheJulianJES) [inline zhaquirks/quirk_ids.py:10]: Okay, the line wrapping is unfortunate, but can't really be avoided unless the name is shortened and/or the comment is shortened.

I'd leave it like this for now and see if this file should be "restructured" (move comments, change to docstrings or whatever) in the future.

**Review by TheJulianJES — APPROVED** [review]
LGTM.
https://github.com/home-assistant/core/pull/101709 should, once a quirk bump is made and merged into HA, be changed to also reference the quirk ID from this file probably.

**Pull request merged**